src/lib/decorators.ts

```diff
@@ -90,6 +90,16 @@ function _query<T>(queryFn: (target: NodeSelector, selector: string) => T) {
   };
 }
 
+// TODO(sorvell): augment TypeScript's EventListenerOptions which does
+// not include these options despite
+// https://github.com/Microsoft/TypeScript/issues/9548
+declare global {
+  interface EventListenerOptions {
```

Comment:
Aren't these already defined in Typescript here?

https://github.com/Microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L33